### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by erikdstock**
*work in progress*
tracked in #17 
This is where I'm at so far- some more major changes I've made are:
- running with babel-node, mainly for server-side es6 modules
- building with yarn
- set up eslint with the basic/'recommended' config
- Added a github widget (and npm dependency, happy to remove)
image

I'd be happy to go forward upgrading the rest of these but since i've already made some big changes I thought i would run it by you first.

### CODE REVIEW

**Comment by davefp** [comment]
Love the progress so far, thanks for your continued work!

The only thing I would like to see at this stage is for the Github widget stuff to be pulled out of this PR. I think extra widgets should eventually be included as modules in a similar way to how [hubot does things](https://github.com/github/hubot/blob/master/docs/scripting.md).

**Comment by erikdstock** [comment]
@davefp - sorry, i've been travelling. updated to remove extra things and focus only on the dependencies (plus eslint...) let me know what you think & thanks.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Handsome is a cousin to [Dashing](http://dashing.io).
 
 ## Prerequisites
 
-You will need *node* and *npm* installed before you can do anything.
+You will need *node* and *npm* and *yarn* installed before you can do anything.
```
Comment (davefp) [inline README.md]: Do we still need `npm` now that we have `yarn`?

Comment (erikdstock) [reply README.md]: fixing!

File: package.json [hunk 2]
```diff
@@ -4,9 +4,11 @@
   "description": "A handsome dashboard framework written in javascript",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "postinstall": "webpack -p --progress --colors",
+    "start": "babel-node app.js",
+    "postinstall": "npm run build",
```
Comment (davefp) [inline package.json:8]: What's the difference between 'build' and 'postinstall' here?

Do we still want to be calling npm?

File: package.json [hunk 3]
```diff
@@ -25,15 +27,19 @@
...
+    "babel-plugin-resolver": "^1.1.0",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "cacheman": "^2.2.1",
     "cacheman-redis": "^1.1.2",
     "css-loader": "^0.23.1",
     "express": "^4.13.4",
     "express-handlebars": "^3.0.0",
+    "github-api": "^3.0.0",
```
Comment (davefp) [inline package.json]: We can get rid of this

**Comment by davefp** [comment]
This all looks good to me. I left a couple of comments on bits I didn't understand. Do you think anything else needs doing before this gets merged?

File: package.json (hunk 2 above)
Comment (erikdstock) [reply package.json:8]: I don't believe there is much of a difference in this case, but I don't know the best way to go about it. I guess my thought was that npm is always there, whereas someone might be choosing to not use yarn (likewise for ci, coveralls, etc).

I also don't *think* there is any difference between yarn vs npm in running tasks- But I could change it if you prefer.

**Comment by erikdstock** [comment]
It's been working fine for me (though i'm not doing anything mission-critical yet)- but I'm removing the github package. I responded to your other comment. Thanks!

**Comment by erikdstock** [comment]
@davefp -> updated to remove the github-api package + clarified the note to make yarn recommended. Thanks!

**Pull request merged**